azure-pipelines.yml

```diff
@@ -57,12 +57,20 @@ jobs:
       toolMajorVersion: V2
       suppressionsFile: './scripts/ci/credscan/CredScanSuppressions.json'
       toolVersionV2: '2.1.17'
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-policheck.PoliCheck@2
+    displayName: 'Run PoliCheck'
+    inputs:
+      targetType: F
```

Comment:
What does `F` mean here?

Reply:
F: Path|Filename - specify root folder | file to be scanned. 
FL:FILE - specify a list of file containing folder | file paths to be scanned.

Reply:
The description is still pretty unclear.